jekyll/_cci2/configuration-reference.md

```diff
@@ -244,7 +249,12 @@ Each `run` declaration represents a new shell. It's possible to specify a multi-
       make test
 ```
 
-Note that our default `shell` has the `-e` option, which causes commands to:
+##### **Shell options**
+**Note** that our default `shell` has a few options enabled by default.
```

Comment:
I think it's a bit strange to put a note immediately after a section header. Were you concerned that people weren't seeing it?